File: app/src/main/java/ph/gov/philfida/da/abacaplanddiseasedeteciontapplayout/otherActivities/SymptomInfo.java
/* Copyright 2019 The TensorFlow Authors. All Rights Reserved.

Licensed under the Apache License, Version 2.0 (the "License");
you may not use this file except in compliance with the License.
You may obtain a copy of the License at

    http://www.apache.org/licenses/LICENSE-2.0

Unless required by applicable law or agreed to in writing, software
distributed under the License is distributed on an "AS IS" BASIS,
WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
See the License for the specific language governing permissions and
limitations under the License.
==============================================================================*/
package ph.gov.philfida.da.abacaplanddiseasedeteciontapplayout.otherActivities;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.widget.TextView;

import ph.gov.philfida.da.abacaplanddiseasedeteciontapplayout.R;

public class SymptomInfo extends AppCompatActivity {
    TextView symptomName,symptomDesc;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_symptom_info);
        assignIDs();
        getExtras();
    }

    private void getExtras() {
        Bundle extras = getIntent().getExtras();
        if (extras != null) {
            String sName = extras.getString("symptomName");
            symptomName.setText(sName);
        }
    }

    private void assignIDs() {
        symptomName = findViewById(R.id.symptomName);
        symptomDesc = findViewById(R.id.symptomDesc);

    }
}

File: app/src/main/java/ph/gov/philfida/da/abacaplanddiseasedeteciontapplayout/otherActivities/SymptomInfo.kt
package ph.gov.philfida.da.abacaplanddiseasedeteciontapplayout.otherActivities

import android.os.Bundle
import android.widget.ImageView
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity
import ph.gov.philfida.da.abacaplanddiseasedeteciontapplayout.R

class SymptomInfo : AppCompatActivity() {
    private lateinit var symptomNameView: TextView
    private lateinit var symptomDescView: TextView
    private lateinit var symptomImageView: ImageView

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_symptom_info)

        symptomNameView = findViewById(R.id.symptomName)
        symptomDescView = findViewById(R.id.symptomDesc)
        symptomImageView = findViewById(R.id.imageView3)

        val symptomName = intent.getStringExtra("symptomName")
        symptomNameView.text = symptomName
        // Description and image can be populated based on symptomName if available in a database
    }
}
